mentat/code_file_manager.py

```diff
@@ -31,13 +31,11 @@ def __init__(self):
 
     def read_file(self, path: Path) -> list[str]:
         session_context = SESSION_CONTEXT.get()
-        git_root = session_context.git_root
 
-        abs_path = path if path.is_absolute() else Path(git_root / path)
-        rel_path = Path(os.path.relpath(abs_path, git_root))
+        abs_path = path if path.is_absolute() else session_context.cwd.joinpath(path)
         with open(abs_path, "r") as f:
             lines = f.read().split("\n")
-        self.file_lines[rel_path] = lines
+        self.file_lines[abs_path] = lines
```

Comment:
Thanks for making this absolute! I thought it was absolute already, but I guess it must've gotten changed somehow (or I'm misremembering)